package.json

```diff
@@ -8,7 +8,11 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "t": ". ./script/bin/env.sh; forge t -vvv",
+    "deploy:local": ". ./script/bin/env.sh; ./script/bin/deploy-local.sh",
+    "compare": "forge snapshot --match-path test/integration/IntegrationTestGasComparisons.sol --snap .dex-comparisons",
+    "compare:check": "yarn compare --check",
+    "compare:csv": "node ./script/dex-comparisons.js"
```

Comment:
@k-xo flagging this change for you, separated the snapshot command from the one that applies `--check` which throws an error in CI if the gas diff changes.